flashinfer/cute_dsl/mla_decode.py

```
@@ -0,0 +1,412 @@
+        f"cute_dsl_mla_decode only supports {_SUPPORTED_DTYPES}, got {query.dtype}"
+    )
+    assert kv_cache.dtype == query.dtype, (
+        f"kv_cache dtype {kv_cache.dtype} must match query dtype {query.dtype}"
+    )
+    B, q_len, H, D_qk = query.shape
+    assert D_qk == kv_lora_rank + qk_rope_head_dim
+    assert kv_lora_rank == _LATENT_DIM
+    assert qk_rope_head_dim == _ROPE_DIM
+
+    q_dtype = query.dtype
+    is_fp8 = q_dtype == torch.float8_e4m3fn
```

Ruff (0.15.5)

[error] Local variable `is_fp8` is assigned to but never used

Remove assignment to unused variable `is_fp8`

(F841)